jenkins/e2e-image/Makefile

```diff
@@ -18,7 +18,7 @@ TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 all: build
 
 kubetest:
-	GOOS=linux GOARCH=amd64 go build -o kubetest ../../kubetest
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o kubetest ../../kubetest
```

Comment:
I looked for a new hotness, and that didn't come up in a cursory google search. :/